.travis.yml

```diff
@@ -9,7 +9,10 @@ go_import_path: github.com/containerd/console
 install:
   - go get -d
   - GOOS=windows go get -d
+  - GOOS=solaris go get -d
 
 script:
   - go test -race
   - GOOS=windows go test
+  - GOOS=solaris go build
+  - GOOS=solaris go test -c
```

Comment:
the travis CI environment is linux based so afaik we can't run solaris binaries or tests there.  even if we could somehow run/test solaris components on linux, as the comments in tc_solaris_nocgo.go indicate, we can't actually cross compile working go binaries for solaris on linux.  lastly, running the tests natively on solaris currently fails.  fixing them will involve more substantial changes that i'll be proposing after this.  i'm keeping that work separate from this PR to try to simplify the review process.